chia/full_node/mempool_manager.py

```diff
@@ -470,6 +470,9 @@ async def validate_spend_bundle(
         if cost == 0:
             return Err.UNKNOWN, None, []
 
+        if cost > self.max_block_clvm_cost:
+            return Err.BLOCK_COST_EXCEEDS_MAX, None, []
```

Comment:
I snuck this check in here. maybe I should have separated this out. In practice, we already have this limit, since it's the max block cost. In tests however, it's really convenient to set the block size smaller

Reply:
marking as resolved